spec/support/features/action_text.rb

```diff
@@ -0,0 +1,23 @@
+module Features
+  if Rails.version >= "6.1"
```

Comment:
Ah, just 0.1 requirements of a version away from doing away with this part :smile: I wonder if we should drop support for 6.* now that they have been EOL'd (https://rubyonrails.org/2024/10/15/new-maintenance-policy-and-eol-annouments). Thoughts, @nickcharlton?

But since this is here and solved for now, I think it's not a blocker. We can decide about Rails requirement separately.

Reply:
This is perhaps a separate discussion, but I wonder if we can drop support post-1.0 (1.1?).

I don't want 1.0 to be so big that it means people avoid upgrading (which is a good argument to call it in the next few weeks?), then drop it on the next release. It's a bit similar to how Apple does it with macOS.